mysite/lti/models.py

```diff
@@ -103,24 +119,29 @@ def enroll(self, roles, course_id):
         course = Course.objects.filter(id=course_id).first()
         if course:
             for role in roles:
-                Role.objects.get_or_create(course=course,
-                                           user=self.django_user,
-                                           role=role)
+                Role.objects.get_or_create(
+                    role=role,
+                    course=course,
+                    user=self.django_user
+                )
 
     def is_enrolled(self, roles, course_id):
         """Check enroll status
 
         :param roles: (str|list)
         :param course_id: int
-        :return: ct.Role
+        :return: bool
         """
         if not isinstance(roles, list):
             roles = roles.split(',')
         course = Course.objects.filter(id=course_id).first()
+        role = Role.INSTRUCTOR if Role.INSTRUCTOR in roles else Role.ENROLLED
```

Comment:
> I'd prefer this to be a conventional if: else: statement rather than a one-line expression

Done in https://github.com/raccoongang/socraticqs2/commit/f69d50f329fb42fdc2718bdb2369788203744139